pom.xml

```diff
@@ -15,6 +15,7 @@
     <modules>
         <module>geoindicators</module>
         <module>prepare_data</module>
+	<module>processing_chain</module>
```

Comment:
The module artifact name should be `processing-chain` with a `-`

Reply:
done

Reply:
then I have renamed the module and folder name as well. But I wonder why we modify the processing_chain name but keep the prepare_data (why don't we replace it by prepare-data ?)

Reply:
It would be replaced later as @ELSW56 was working on it.